ros_ws/src/controller/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.8)
+project(controller)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(std_msgs REQUIRED)
```

Comment:
Are these all ROS-based dependencies? I.e. not something you wanted to add?

Reply:
Yes they are ros dependencies. Each one has it's functionality. Note, std_msgs is a wrapper around primitive type and we will most likely build our own.